test/promql/tests/data_plane/pipeline_to_snapshot_test.yaml

```diff
@@ -0,0 +1,108 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.pipeline_to_snapshot_alerts.yaml
+
+tests:
+
+ # Scenario when alert should NOT trigger
+  - interval: 1m
+    input_series:
+      - series: 'integration_svc_response_seconds_bucket{le="30"}'
+        values: '0+95x1'  # 95 occurrances took less than 30s
+      - series: 'integration_svc_response_seconds_bucket{le="+Inf"}'
+        values: '0+100x1'  # 100 total occurrences
```

Comment:
This test case is providing only 2-min-worth of measurements while the rule is applied on a time frame of 5 minutes + 1 minute of waiting. We need some more samples  and later eval_time for this one, or to have another test case with those.